includes/admin/payments/class-payments-table.php

```diff
@@ -935,14 +935,14 @@ public function get_payment_counts() {
 	public function payments_data() {
```

Comment:
I am not sure on what basis we are sanitizing request data because I can see in this file few other request data in other function is not sanitized.  

@JasonTheAdams Can we discuss this today?

![image](https://user-images.githubusercontent.com/1784821/116512115-282dcf00-a8e5-11eb-9f72-a198d1ed9759.png)
![image](https://user-images.githubusercontent.com/1784821/116512176-3da2f900-a8e5-11eb-8dc2-7044fca8b855.png)
![image](https://user-images.githubusercontent.com/1784821/116512265-62976c00-a8e5-11eb-90f0-e4e6f90b8493.png)